modules/performanceplatform/manifests/checks/celery_worker.pp

```diff
@@ -0,0 +1,16 @@
+class performanceplatform::checks::celeryworker() {
+  sensu::check { 'celery_worker_is_down':
+      command  => "/etc/sensu/community-plugins/plugins/processes/check-procs.rb -p 'celery' -C 1 -W 1",
+      interval => 60,
+      handlers => ['default']
+  }
+
+  performanceplatform::checks::graphite { "check_celery_worker_error_rate":
+      target   => "divideSeries(hitcount(stats.pp.apps.backdrop.transformers.worker.run_transform.error,'1hr'), hitcount(stats.pp.apps.backdrop.transformers.worker.run_transform.success,'1hr'))"
```

Comment:
Might be an idea to comment this so we have some idea what it does in the future. I'm already a little confused what it is doing!